### ISSUE TITLE
Staff not making sense

### ISSUE THREAD

**Issue by PSIPowerhouse** [sm64js/sm64js#769]
Simple as that, the rules are completely different from what i'm seeing. Specifically Agent X is the problem. Yesterday i was banned for "Toxicity".  The dumbest part is the fact that i was being the LEAST toxic of everyone there. It all started in the discord. I joined in the middle of someones conversation with Agent X but i didn't know because I didn't see the messages from or directed towards him. And then somebody else said "Nobody Asked and I'm sure X doesn't care either". I told them that i wasn't talking to them, and they made a huge ass argument for no reason. A little while after i go back to sm64js and Mr Yahoo and Some others start being assholes and attacking me. I said "Please don't start this shit" and Mr Yahoo was like "Yes i am gonna start this shit" apparently i did something to them on discord or something? idk what i even did to them but they wouldn't tell me. So then some time goes by as people repetitively attack me as i tell them to stop and piss off. Idk what i did to most of them but i know what i  did to ONE of them. "Pre each123" or something. I was doing my normal speedrun movement around, you know, Slide kicking, Dives, Jump dives, etc., and i accidentally hit him (Same thing also happened with this kid named laki a month or so ago). Then after that I guess he teamed with Yahoo and started attacking me. A couple minutes go by and Agent X comes in saying "you wanna keep trolling suru or ban" everyone said ban including me because i was tired of it. And it said i was "banned for toxicity" I RARELY SAID ANY FUCKING THING!!! If anything THEY, INCLUDING AGENT X, should've been the ones banned, NOT me. Because THEY were the ones talking all shit about me. I personally didn't give a damn about them talking shit, but the bottom line is the fact that i shouldn't have been the one that got banned for toxicity if i wasn't hardly even talking. And they should've

**Comment by Someone567123** [comment]
yeah this game has had shit moderation since I left

**Comment by AgentXLP** [comment]
![image](https://user-images.githubusercontent.com/44549182/204666592-e8ba1e76-a7c7-4c60-a52b-49f08622df1a.png)
I'd disagree Someone, as you were able to talk about shit like this with no repercussions

**Comment by Someone567123** [comment]
bro I was such a rascal back then! what a funny screenshit

**Comment by AgentXLP** [comment]
![image](https://user-images.githubusercontent.com/44549182/204667047-d331521f-92d6-4556-8b0f-a5ea1d25f1b3.png)
![image](https://user-images.githubusercontent.com/44549182/204667073-bb927a82-46fd-4a18-adc2-a43e128faf47.png)
![image](https://user-images.githubusercontent.com/44549182/204667099-a880583e-af2c-4fd0-8723-049135aeae36.png)
![image](https://user-images.githubusercontent.com/44549182/204667123-d6d88acd-73c7-4e2e-b38b-bb694b547d52.png)
![image](https://user-images.githubusercontent.com/44549182/204667154-4b058ffa-7569-4208-bd00-cf24e0475177.png)
The points you try to make are quite hypocritical in the face of what had actually happened, if you take a look at the attached screenshots you'll see you were in fact being the asshole and you fully knew that which puts you in no position to claim you were the least toxic. You were banned from the discord for rule 13, you had only joined with the intent to be a dick to everyone and your ban is completely justified and fair. In-game doesn't have the exact same rules as on the discord server, notably the age requirement on discord to abide with it's TOS. When you start to talk about in-game what you say comes off with major victim complex vibes as you paint the messages sent as a genuine plea, however knowing you this likely isn't the case.
![image](https://user-images.githubusercontent.com/44549182/204667973-49e5f199-aab7-4ee3-8926-533efc606157.png)
![image](https://user-images.githubusercontent.com/44549182/204668035-7ca5321f-2440-49bc-811d-e8ec3051294d.png)
To add onto that, you were going to receive punishment regardless of what people had to say, by saying "yall wanna keep trolling suru or ban" I wanted to gage the general opinion of everyone in-game and it was mostly negative, I had decided somewhere between several hours to 2 days and weighed it out to a 1 day ban. You said quite a lot actually during your time being a dick prior to posting this issue, there's an entire chat log in the discord of things you've said in case you haven't realized that yet and if you want I can pull up many instances of you being toxic, but here are a few anyway:
![image](https://user-images.githubusercontent.com/44549182/204668753-6699c9e9-16db-40b2-8099-5ab96447a40d.png)
![image](https://user-images.githubusercontent.com/44549182/204668785-fed4a879-2dfd-4c13-8550-692f77a75546.png)
![image](https://user-images.githubusercontent.com/44549182/204668862-e1e78ba6-793e-4fbc-9b00-4e0661237bef.png)
![image](https://user-images.githubusercontent.com/44549182/204669003-4d57c0b9-5218-40d2-8189-2462d316c356.png)
In summary, you paint yourself to be the poor innocent victim of evil trolls on mario 64 javascript when in reality it's a whole other story.

**Comment by Someone567123** [comment]
in other words: L

**Comment by PSIPowerhouse** [comment]
First of all, all i said was Motherfucker, I didn't join with the intent to "be a dick", because i joined about 2 weeks earlier. I just never said anything on the server. Second, How the hell am i breaking discords Age requirement TOS? Explain that shit.

>"yall wanna keep trolling suru or ban" I wanted to gage the general opinion of everyone in-game

Again, i never said hardly anything besides "shut the fuck up" and "I don't care anymore". So what was there to ban me for exactly?

>if you want I can pull up many instances of you being toxic

What the hell does that have anything to do with why i was banned YESTERDAY for being "Toxic". Also, Me saying "Fuck dnaleeL, Fuck nailuJ, Fuck all of them" is me saying that shit is really annoying, not me being Toxic. 

Also, Everything Past "if you want I can pull up many instances of you being toxic" is no longer valid. That was 2 weeks ago.

>you paint yourself to be the poor innocent victim of evil trolls on mario 64 javascript when in reality it's a whole other story

I said that THEY were talking shit about ME! How the hell do you NOT understand what that means? You were THERE when they kept doing it. I swear you're the worst moderator ever. How the hell do you ban someone who rarely said anything, whereas 7 other people on the other hand, are talking all the shit? Make that make sense.

**Comment by Bloxxel64** [comment]
toxic
1 of 3
[adjective](https://www.merriam-webster.com/dictionary/adjective)
tox·​ic [ˈtäk-sik ](https://www.merriam-webster.com/dictionary/toxic?pronunciation&lang=en_us&dir=t&file=toxic001)

3
: extremely harsh, malicious, or harmful
toxic sarcasm

**Comment by Bloxxel64** [comment]
You were making rude comments and lashing out for no reason, explain how that isn't toxic.

**Comment by Bloxxel64** [comment]
In fact you still are.

**Comment by AgentXLP** [comment]
> First of all, all i said was Motherfucker, I didn't join with the intent to "be a dick", because i joined about 2 weeks earlier. 

That doesn't invalidate rule 13 at all, you joined and did nothing but start drama.

> How the hell am i breaking discords Age requirement TOS? Explain that shit.

You aren't, simply an example I brought up to show the contrast between the discord's rules and in-game rules

> is no longer valid. That was 2 weeks ago.

A mere 2 weeks doesn't invalidate anything I said, if it were 2 years then yeah whatever you've probably changed but that simply just doesn't happen in a matter of 2 weeks PSI.

> I said that THEY were talking shit about ME! How the hell do you NOT understand what that means? You were THERE when they kept doing it. I swear you're the worst moderator ever. How the hell do you ban someone who rarely said anything, whereas 7 other people on the other hand, are talking all the shit? Make that make sense.

They were talking shit about YOU because you were talking shit about THEM in the first place as well as being annoying, toxic and the self proclaimed "asshole of sm64js" chief, I'm not arguing about this any further and instead of being the bigger person in this situation you took to github to whine about your bans instead of waiting for the in-game one to end and asking to get in contact with any of us, github issues are for bug reporting with the game or suggestions, not to complain about staff because you got punishment for being toxic.

**Comment by Someone567123** [comment]
hey wait a minute, that's from a year ago
